src/monitor-qualification-workspace/monitor-qualification.service.ts

```diff
@@ -52,8 +55,24 @@ export class MonitorQualificationWorkspaceService {
                 qualificationRecord.id,
                 qualification,
               );
+              this.pctQualificationService.importPCTQualification(
```

Comment:
We will need to add this to a list of promises, and then await those promises.all